cpp/oneapi/dal/algo/rbf_kernel/BUILD

```diff
@@ -17,17 +17,33 @@ dal_module(
 
```

Comment:
It's not PCA related cahnge

Reply:
I've added support for DPC++ tests in Bazel to be able to run DPC++ PCA tests and had to fix existing rules/tests.